package.json

```diff
@@ -33,14 +32,22 @@
     "mocha": "^3.1.2",
     "phantomjs-prebuilt": "^2.1.4",
     "sinon": "^1.17.2",
-    "webpack": "^2.2.1"
+    "uglifyjs-webpack-plugin": "^1.2.4",
+    "webpack": "^4.5.0",
+    "webpack-bundle-analyzer": "^2.11.1",
+    "webpack-cli": "^2.0.14"
   },
   "scripts": {
     "build": "npm run build-debug && npm run build-min",
-    "build-debug": "webpack",
-    "build-min": "cross-env NODE_ENV=production webpack --optimize-minimize",
+    "build-debug": "webpack --mode=development",
+    "build-analyze": "cross-env ANALYZE_BUIlD=true npm run prepublish",
```

Comment:
Typo also here `ANALYZE_BUILD`

Good job on this one 👍 Hopefully it will be merged 🎉

Reply:
Thanks